soh/soh/Enhancements/audio/AudioEditor.cpp

```diff
@@ -306,19 +305,11 @@ void DrawTypeChip(SeqType type) {
     ImGui::EndDisabled();
 }
 
-void DrawSfxEditor(bool& open) {
-    if (!open) {
```

Comment:
Just so I'm understanding, we're completely moving away from relying on cvars for open/closed window state?

I thought this was a code smell anyway trying to keep this and ImGui in sync, _but_ does this mean we lose our ability to persist window state after restarts? (I just made a recent effort to get this working across all windows)

Reply:
we are not, the constructor for `GuiElement` now takes in `visibilityConsoleVariable`

https://github.com/Kenix3/libultraship/blob/6d29ff2dbc4507b59771d5af84cb58eaacd2f9a7/src/window/gui/GuiElement.cpp#L6-L12

and the logic for if we should draw now lives in that base class

https://github.com/Kenix3/libultraship/blob/6d29ff2dbc4507b59771d5af84cb58eaacd2f9a7/src/window/gui/GuiElement.cpp#L35-L43

Reply:
Aha I see. And those are all listed in `soh/soh/SohGui.cpp`?

Reply:
Yes, it's defined in the constructor.